src/js/views/Certificates/layout/Cards.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+
+import { Box, Grid, IconButton, Tooltip, Typography } from '@material-ui/core';
+import { Check, Close, CertificatesOther, Star, StarBorderOutlined } from '@material-ui/icons';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import DataCard from '../../../common/components/Cards/DataCard';
+import { useCardsStyles } from './style';
+
+const Cards = ({
```

Comment:
You should remove everything related to devices, such as favorite buttons, attrs, and use this component in the `certificates/View.jsx` component.